suclass/lms/templates/header.html

```diff
@@ -12,18 +12,16 @@
   <img src="${static.url('images/stanford-S-logo.png')}" height="35" alt="Home Page" />
```

Comment:
Thanks to comp theming, we may be able to unfork this block.

The template tries to pull the logo from a number of locations first,
but falls back to `images/logo.png`.
I think if we renamed the file to the above, it should be found automatically.